vale/dictionaries/en_US-grafana.dic

```diff
@@ -1,4 +1,4 @@
-85
+86
```

Comment:
I'm interested in this value. I've tried poking the Vale docs, but not sure what to search for. I can see it's a count of entries in this file.. but for what purpose? What happens if we forget to increment this value?

Reply:
Ah yeah, this is not actually a Vale thing but rather an aspect of the Hunspell dictionary format. 

The best resource I have right now is https://manpages.ubuntu.com/manpages/focal/man5/hunspell.5.html.

The need to count the entries is actually why I have a separate `.wordlist` file that we can just add entries to and then run `make all` which generates the `.dic` file from the `.wordlist` file so the number is always right.